### PR TITLE
Update custom-elements.json

### PR DESCRIPTION
**Pull request by jogibear9988**
fix to match spec

### CODE REVIEW

File: custom-elements.json [hunk 1]
```diff
@@ -1,19 +1,41 @@
...
+  "schemaVersion": "1.0.0",
+  "readme": "README.md",
+  "modules": [
     {
-      "name": "paint-app",
-      "description": "An open-source, Web Components-based remake of MS Paint using modern web capabilities.",
-      "properties": [],
-      "events": [
+      "kind": "javascript-module",
+      "path": "index.js",
```
Comment (christianliebel) [inline custom-elements.json:7]:
```suggestion
      "path": "dist/elements/index.js",
```

File: custom-elements.json [hunk 2]
```diff
@@ -1,19 +1,41 @@
...
       ],
-      "slots": [],
-      "cssProperties": []
+      "exports": [
+        {
+          "kind": "custom-element-definition",
+          "name": "paint-app",
+          "declaration": {
+            "name": "App",
+            "module": "index.js"
```
Comment (christianliebel) [inline custom-elements.json:35]:
```suggestion
            "module": "dist/elements/index.js"
```

**Review by christianliebel — CHANGES_REQUESTED** [review]
I guess this needs to relate to the build location in the final package?

**Comment by jogibear9988** [comment]
I think the index.js would also work, cause it exports everything. But your index is in the dist dir.... my fault.

**Comment by jogibear9988** [comment]
I'm currently workin on a manifest loader for my designer, so I'm looking for components using the manifest, and try to fix :-)

**Review by christianliebel — APPROVED** [review]

**Comment by christianliebel** [comment]
LGTM, could you please squash your commits and rename the final commit to something like `fix(custom-elements): update manifest to match spec`? I'm using [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) here. Thanks for your support!

**Comment by jogibear9988** [comment]
@christianliebel 
squashed and renamed.

could you create a new npm when you merge? so I could test if it works?

**Comment by christianliebel** [comment]
> @christianliebel squashed and renamed.
> 
> could you create a new npm when you merge? so I could test if it works?

Done!

**Comment by jogibear9988** [comment]
thx

**Pull request merged**